src/components/button/button.module.scss

```diff
@@ -1,80 +1,85 @@
 @import '../../styles/variables.module.scss';
 
-.buttonWrapper {
-  display: inline-block;
-  &.disabled {
-    opacity: 0.4;
-    pointer-events: none;
-  }
-
-  &:hover {
-    transform: scale(1.02);
-  }
-}
-
 .button {
   background-color: transparent;
-  display: flex;
-  align-items: center;
-  justify-content: center;
+  position: relative;
   color: $button-color;
   border-width: 1px;
   border-style: solid;
   outline: none;
   cursor: pointer;
 
-  &.primary {
-    border-image-source: $primary-border;
-    border-image-slice: 1;
+  &:disabled {
+    opacity: 0.4;
+    pointer-events: none;
   }
 
-  &.secondary {
-    border-color: $secondary-border;
+  &:hover {
+    transform: scale(1.02);
   }
+}
 
-  &.normal {
-    font-size: $text-small;
-    line-height: $lh-small;
-    padding: 17px $space-lg;
+.primary {
+  border-image-source: $primary-border;
+  border-image-slice: 1;
 
-    @media (max-width: $max-md) {
-      padding: $space-md;
-    }
+  &::after {
+    background: $primary-border;
   }
+}
 
-  &.large {
-    font-size: $text-normal;
-    line-height: 22px;
-    padding: 20px 44px;
+.secondary {
+  border-color: $secondary-border;
 
-    @media (max-width: $max-md) {
-      font-size: $text-small;
-      padding: $space-md $space-xl;
-    }
+  &::after {
```

Comment:
Don't you want `content: ''` here as well?

Reply:
No need, all the common properties for primary and secondary are declared together here:
```scss
.primary,
.secondary {
  margin-bottom: 5px;

  &::after {
    content: '';
    height: 1px;
    width: calc(100% + 2px);
    position: absolute;
    bottom: -6px;
    left: -1px;
  }
}
```